docs/setup/native.md

```diff
@@ -160,3 +162,4 @@ All of the OSX instructions assume `homebrew` is your [package manager](https://
    - `sudo apt install -y clamav`
 1. Install pdftk
    - `sudo apt install -y pdftk`
+1. continue with [Base setup](README.md#base-setup)
```

Comment:
For the "vanilla" installs (`redis`, `imagemagick`, `poppler`), rather than have them in 3 separate steps, can we just make one step and say: 


1. Install `redis`, `imagemagic`, & `poppler`:
```
brew install redis imagemagick poppler
```

Seems more brief / less reading.